src/content/app/genome-browser/components/drawer/drawer-views/variant-summary/variant-consequence/VariantConsequence.tsx

```diff
@@ -24,10 +24,11 @@ import styles from './VariantConsequence.scss';
 
 type Props = {
   variant: Pick<Variant, 'prediction_results'>;
+  showColour?: boolean;
```

Comment:
`showColour` -> `withColour`?

Alternatively, we can export the `getMostSevereVariantConsequence` function from it, and just use it directly in `VariantSummaryStrip`.

Reply:
✅